microsoft-365/enterprise/modern-web-part-optimization.md

```diff
@@ -51,13 +51,16 @@ If the **Web parts are impacting page load time** result appears in either the *
 
 Information available in the results includes:
 
-- **Made by** shows whether the web part is custom or Microsoft OOTB
-- **Name and ID** shows identifying information that can help you find the web part on the page
-- **Total** shows the total time for the web part to load
-- **Module Load** shows the time taken to fetch and load the web part components
-- **Lazy Load** shows the time for deferred loading of web parts not seen in the main section of the page
-- **Init** shows the time taken for web part initialization
-- **Render** shows the time taken for the web part to fetch and render results
+- **Made by** shows whether the web part is custom or Microsoft OOTB.
+- **Name and ID** shows identifying information that can help you find the web part on the page.
+- **Total** shows the total time for the web part to module load, initialize and render. It is the total relative time taken by the web part to render on the page, from beginning to the end.
+- **Module Load** shows the time taken to download, evaluate and load the extensions JavaScript and CSS files. It will then start the Init process.
```

Comment:
```suggestion
- **Module Load** shows the time taken to download, evaluate, and load the extension's JavaScript and CSS files. It will then start the Init process.
```